client/get.go

```diff
@@ -50,9 +64,11 @@ func runGet(cmd *Command, args []string) *ErrorStatus {
 	if *getTinkKeyset {
 		tinkKeysetInBytes, err := retrieveTinkKeyset(keyID, *getNetwork)
 		if err != nil {
+			failureGetKeyMetric(keyID)
 			return err
 		}
 		fmt.Printf("%s", string(tinkKeysetInBytes))
+		successGetKeyMetric(keyID)
 		return nil
 	}
 	if *getTinkKeysetInfo {
```

Comment:
metrics not sent for this branch?

Reply:
thanks, added metrics here. I originally thought this was only a metadata code path but looks like it also retrieves the key from Knox so it should have metrics.